src/main/webapp/js/controllers/techDetailsController.js

```diff
@@ -569,5 +570,17 @@ angular.module('techGallery').controller(
       });
     }
     
+    $scope.editTechnology = function(){
+    	window.location = $scope.redirectUrl($scope.id, '/createTech');
+    }
+    
+    $scope.redirectUrl = function(techId, servlet) {
+        var protocol = location.protocol + '//';
```

Comment:
@tsr-thulio this is the same method as techListController.js:52
What do you think about refatoring it to an util file?